external/easylogging++/easylogging++.cc

```diff
@@ -1977,9 +1977,9 @@ void VRegistry::setCategories(const char* categories, bool clear) {
   }
   if (!m_categoriesString.empty())
     m_categoriesString += ",";
```

Comment:
Shouldn't we add the `,` only if categories isn't empty too? Otherwise it would add a useless `,` at the end of m_categoriesString.